docs/_docs/collections.md

```diff
@@ -51,7 +51,7 @@ defaults:
 
 Create a corresponding folder (for example, `<source>/_my_collection`) and add
 documents. YAML Front Matter is read in as data if it exists, and everything
-after it is available in the document's `content` attribute. If no YAML Front
+after it is accessible via the document's `content` attribute. If no YAML Front
```

Comment:
sorry, i just don't see the logic there. if you were omitting words, then the comma could work. Such as, "YAML Front Matter is read in as data if it exists, and everything
after, in the Document's `content` attribute." In that case, the "," stands for "is read in as." 

There is also a slight pause in your phrasing, so I can see why you might want a comma there to indicate the pause. 

I think the overall sentence is a little janky anyway. The problem is that "it" is ambiguous (does it refer to Front Matter or data -- at first read, it's not all that clear), and there are 2 "it" references. 

Here's how I would fix that:

> Create a corresponding folder (e.g. `<source>/_my_collection`) and add
documents. YAML front matter is processed if the front matter exists, and everything
after the front matter is pushed into the Document's `content` attribute. If no YAML Front
Matter is provided, Jekyll will not generate the file in your collection.

I think "read in as data" is kind of vague, and "is stuck into" is also an odd word choice. Do you like this new paragraph better, or should I keep it as is?

I don't mean to bikeshed this point. Probably doesn't matter a whole lot either way. I'll commit this update and you can downvote it if you hate it.